### PR TITLE
Schema adjusted to make products table plural, add seed script

### PR DESCRIPTION
**Pull request by timothydadams**

### CODE REVIEW

**Comment by timothydadams** [comment]
@jayrbarr @JasperC1999 seed file complete, slight change to schema for handling decimal places, turns out the first num in the decimal data type is for specifying the TOTAL length of a float number instead of just the digits before the '.'

File: db/dbSeed.js [hunk 1]
```diff
@@ -1,7 +1,145 @@
...
+  database: 'fec_Module_1'
+});
 
+connection.connect((err) => {
+  if (err) {
+    console.log('Make sure you load the db script in mysql before you try to connect', err);
+    return;
+  }
+  console.log('Successfully connected to the database!');
+});
```
Comment (JasperC1999) [inline db/dbSeed.js:15]: Is it necessary to connect the db in the seed?

File: db/dbSeed.js [hunk 2]
```diff
@@ -1,7 +1,145 @@
...
+};
+
+
+//call reset functions
+setUpProductTable();
+setUpImagesTable();
+setUpESRBRatingsTable();
+
+//end connection
+connection.end();
```
Comment (jayrbarr) [inline db/dbSeed.js:145]: Both git and airbnb are gonna harass you about needing a newline at the end of every script.

Comment (timothydadams) [reply db/dbSeed.js:145]: I need to get Airbnb style guide going.

File: db/dbSeed.js (hunk 1 above)
Comment (timothydadams) [reply db/dbSeed.js:15]: I guess I could just import the db/index.js file.

I just wanted to make sure that connection to the db was killed after seeding w/o impacting the actual server access to the db so I made it a separate instance.

**Pull request merged**